tests/Unleash.Tests/Communication/MockHttpClient.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Unleash.Communication;
+using Unleash.Internal;
+using Unleash.Serialization;
+using RichardSzalay.MockHttp;
+
+namespace Unleash.Tests.Communication
+{
+    internal static class MockHttpClient
+    {
+        internal static Tuple<MockHttpMessageHandler, UnleashApiClient> MakeMockClient(string url)
```

Comment:
It's 2024, we can just call functions rather than relying on an inheritance chain for test setup